### ISSUE TITLE
Text overflowing the page on Alaveteli admin dashboard

### ISSUE THREAD

**Issue by MattK1234** [mysociety/alaveteli#6101]
The requests under the tab named "Review requests reported by users as requiring administrator attention" overflow the end of the page when there is a lengthier user message added. You also then cannot see the date of the report as its pushed off the end.

If you look under "Review requests marked by the requester as requiring administrator attention" this is not the case, the reports here are correctly formatted.

Here's an example:
![image](https://user-images.githubusercontent.com/47503358/107126035-4d421100-68a5-11eb-94bb-943819807943.png)

**Comment by garethrees** [comment]
I can't seem to reproduce this. 🤔

<img width="1020" alt="Screenshot 2021-02-08 at 09 09 11" src="https://user-images.githubusercontent.com/282788/107198683-5807bd80-69ed-11eb-8dd3-d3b105c6a56b.png">

Could you paste the text you used to cause the issue?

**Comment by MattK1234** [comment]
The issue I setup was a fake report so that I could post it publicly. The text is at https://groups.google.com/a/mysociety.org/g/alaveteli-pro/c/FOxuQ9gSi5c/m/QocnHVhLAgAJ

I had noticed this with actual reports on WDTK before I created the fake one. They've since been actioned and at the moment it's not doing it (I believe because there is not enough text in the reports that are there).

I've just tried making another report with the same text and it's not doing it this time. It might have been related that there were a lot of reports (circa 40 or 50) at the time I noticed it.
